### DISCUSSION TITLE
ran `./install.sh -r` to update hyprdots to latest config and lost vscode data ( extensions )

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [prasanthrangan/hyprdots#319] [Q&A] [answered]
I followed this https://github.com/prasanthrangan/hyprdots/discussions/262 and thought it would update hyprdots to latest changes in git. I opened Vscode and lost all my extensions and my user settings. 

Anyway to fix this?

**Comment by kRHYME7** [comment, marked as answer]
Please Report an issue, but You can retrieve your latest back up at ~/.config/cfg* 

copy the folder name ``` Code``` and paste it to your ~/.config

**Comment by solitudealma** [comment]
you can change the rule in restore_cfg.lst, like: N|Y|${HOME}/.config/Code/User|settings.json|visual-studio-code-bin

**Comment by kRHYME7** [comment]
My response here is outdated but here's an explanation on some flags as of writing this 

https://github.com/kRHYME7/Hyprdots-ctl/issues/11#issuecomment-1913179843
>YY: Both flags are set to 'Y'.
In this case, the script will move the existing configuration to the backup directory (overwriting it) and then restore the new configuration from the clone directory.
NN: Neither flag is set to 'Y'.
The script will copy the existing configuration to the backup directory (preserving it) and then restore the new configuration from the clone directory.
YN: The overwrite flag is set to 'Y' and the backup flag is not set to 'Y'.
The script will move the existing configuration to the backup directory (overwriting it) and then restore the new configuration from the clone directory.
NY: The overwrite flag is not set to 'Y' and the backup flag is set to 'Y'.
The script will copy the existing configuration to the backup directory (preserving it) and then restore the new configuration from the clone directory.
